src/main/java/appeng/integration/modules/RF.java

```diff
@@ -69,7 +69,7 @@ public void postInit() {
         this.registerRFAttunement("ThermalDynamics", "ThermalDynamics_0", 6);
 
         this.registerRFAttunement("EnderIO", "itemPowerConduit", OreDictionary.WILDCARD_VALUE);
-        this.registerRFAttunement("EnderIO", "blockCapacitorBank", 0);
+        this.registerRFAttunement("EnderIO", "blockCapBank", 0);
         this.registerRFAttunement("EnderIO", "blockPowerMonitor", 0);
     }
```

Comment:
You should also add this to register the newer (iron, melodic, silver, etc) energy conduits:
```suggestion
        this.registerRFAttunement("EnderIO", "itemPowerConduitEndergy", OreDictionary.WILDCARD_VALUE);
    }
```